resource_tracker/api/resource_api_view.py

```diff
@@ -42,9 +42,13 @@ class ResourceGetDelete(generics.RetrieveUpdateDestroyAPIView):
     permission_classes = [IsAdminUser]
     serializer_class = ResourceSerializer
 
+    def get_serializer(self, *args, **kwargs):
+        if 'data' in kwargs:
+            kwargs['data']['resource_group'] = self.kwargs.get('resource_group_id', None)
+        return super(ResourceGetDelete, self).get_serializer(*args, **kwargs)
+
```

Comment:
what if "data" not in kwargs?
Why do we add a method here without removing some code in the serializer?
It was working without this part.

Reply:
We used a generic API view, data is sent for update (patch and put), and resource_group is a mandatory field now.